diff --git a/node_modules/@types/leaflet.locatecontrol/index.d.ts b/node_modules/@types/leaflet.locatecontrol/index.d.ts
index 0afcd0d..0f57e97 100644
--- a/node_modules/@types/leaflet.locatecontrol/index.d.ts
+++ b/node_modules/@types/leaflet.locatecontrol/index.d.ts
@@ -38,7 +38,8 @@ declare module 'leaflet' {
             onLocationOutsideMapBounds?: any;
             showPopup?: boolean;
             strings?: any;
-            locateOptions?: L.LocateOptions;
+            getLocationBounds?: Function;
+	          locateOptions?: L.LocateOptions;
         }
     }
 
